app/components/nav.jsx

```diff
@@ -75,6 +86,10 @@ export class Nav extends Component {
     return (
       <nav>
         <Link to="/">Pokédex Tracker</Link>
+        <a onClick={this.toggleNightMode} className="tooltip tooltip-below">
+          <i className="fa fa-adjust" />
+          <span className="tooltip-text">Night Mode {nightMode ? 'Off' : 'On'}</span>
+        </a>
```

Comment:
would be nice if we could move these 3 elements (home link, night mode toggle, and patreon link) to a shared var so we don't have to repeat them twice -- but i can't do that because it has to be enclosed as 1 element. wrapping it in a div messes up the styling, and we don't have fragments. thoughts?

Reply:
updated to use a fragment!